simplecal/README.md

```diff
@@ -0,0 +1,45 @@
+# simplacal
+
+Simplecal is extremely easy way how to show the calendar applet after clicking on time blocklet
+
+![](screen.jpg?raw=true)
+
+
+# Requirements
+
+Dependencies:  bash, calendar applet
+
+Suggested: fonts-font-awesome
+ 
+# i3blocks sutup
+
+```
+[time]
+#label=
+label=time:
+command=$SCRIPT_DIR/simplecal
+interval=5
+
+```
+
+# Calendar setup
+
+I chose [Gsimplecal](https://github.com/dmedvinsky/gsimplecal)
```

Comment:
Avoid reference to yourself. I would say

The script uses Gsimplecal, available in your distribution's package repositories or [here](https://github.com/dmedvinsky/gsimplecal). If you prefer a different calendar applet, specify it in the blocklet config with `instance=`.